Assignment3/getMiddleCharacter.js

```diff
@@ -0,0 +1,17 @@
+function getMiddle(stringValue)
+{

```

Comment:
Can u google if there are better "getMiddle" methods?

Reply:
ok I will do it